mario-less/mario.c

```diff
@@ -1,12 +1,18 @@
-#include <cs50.h>
+// #include <cs50.h>
 #include <stdio.h>
 
 int main(void)
 {
     int height, row, column, space;
     do
     {
-        height = get_int ("Enter height: ");
+				printf("Enter Height: ");
+				scanf("%d", &height);
+				if (height > 8) {
+					printf("Please enter height less than 8.\n");
```

Comment:
Make this less than or equal to
```suggestion
					printf("Please enter height less than or equal to 8.\n");
```